pkgs/tools/networking/doggo/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  runCommand,
+  installShellFiles,
+}: let
+  buildDate = "2022-08-15T21:36:49";
+  buildCommit = "2cf9e7b";
+in
+  with lib;
+    buildGoModule rec {
+      pname = "doggo";
+      version = "0.5.4";
+      src = fetchFromGitHub {
+        owner = "mr-karan";
+        repo = pname;
+        rev = "v${version}";
+        sha256 = "sha256-6jNs8vigrwKk47Voe42J9QYMTP7KnNAtJ5vFZTUW680=";
+      };
+
+      vendorSha256 = "sha256-pyzu89HDFrMQqYJZC2vdqzOc6PiAbqhaTgYakmN0qj8=";
+      nativeBuildInputs = [installShellFiles];
+      subPackages = ["cmd/doggo"];
+
+      ldflags = [
+        "-w -s"
+        "-X main.buildVersion=${buildCommit}"
+        "-X main.buildDate=${buildDate}"
+      ];
+
+      postInstall = ''
+        installShellCompletion --cmd doggo --fish --name doggo.fish completions/doggo.fish
+        installShellCompletion --cmd doggo --zsh --name _doggo completions/doggo.zsh
+      '';
+
+      meta = {
+        homepage = "https://github.com/mr-karan/doggo";
+        description = "Command-line DNS Client for Humans. Written in Golang";
+        longDescription = ''
+          doggo is a modern command-line DNS client (like dig) written in Golang.
+          It outputs information in a neat concise manner and supports protocols like DoH, DoT, DoQ, and DNSCrypt as well
+        '';
+        license = licenses.gpl3Only;
```

Comment:
Upstream includes [LICENSE](https://github.com/mr-karan/doggo/blob/v0.5.4/LICENSE) which seems to be GPL 3.0, but the [readme](https://github.com/mr-karan/doggo/blob/v0.5.4/README.md) has a notice for apgl3Plus: 

> This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

My guess is they wanted gpl3Plus, but that's a *guess*, and maybe someone should ask them. Other than that, I don't have any specific suggestions for changing this line.

Reply:
Yes good point, I asked the repo owner.